File: Team_Workspace/Abdelrahman-Adel/Tasks/The Universal Printer/task_18.py
from typing import Any

def log_info(data: Any):
    print(f"[SYSTEM LOG]: {data}")
robot_id = 101
log_info(robot_id)

battery = 12.5
log_info(battery)

status = "Ready"
log_info(status)

values = [1, 2]
log_info(values)

File: Team_Workspace/Fatma_Ahmed/cpp_tasks/task18/task18.cpp
// task18.cpp : This file contains the 'main' function. Program execution begins and ends there.
//

#include <iostream>
using namespace std;

template <typename T>
void logInfo(T arg)
{
    cout << "[SYSTEM LOG]: " << arg << endl;

}
int main()
{
    int id = 101;
    double battery =12.5;

    string status = "Ready";
    logInfo<int>(id);
    logInfo<double>(battery);
    logInfo<string>(status);

}

